docs/src/main/asciidoc/aws-lambda.adoc

```diff
@@ -297,6 +300,23 @@ dependencies {
     testImplementation 'io.rest-assured:rest-assured'
 }
 ----
+****
+
+[role="secondary asciidoc-tabs-sync-kotlin"]
+.Kotlin DSL
+****
+[source,kotlin]
+----
+dependencies {
+    implementation(enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}"))
+    implementation("io.quarkus:quarkus-resteasy")
+    implementation("io.quarkus:quarkus-amazon-lambda")
+
+    testImplementation("io.quarkus:quarkus-junit5")
+    testImplementation("io.rest-assured:rest-assured")
+}
```

Comment:
In this particular example, can't we find a way to make the same content work for both?

I remember we adjusted what we have here for the extension https://quarkus.io/guides/validation#creating-the-maven-project to try to make it work for both.